src/GraphQL.Tests/Types/NonNullGraphTypeTests.cs

```diff
@@ -94,34 +94,35 @@ public NullableSchema()
         var query = new ObjectGraphType();
 
         query.Field<NullableSchemaType>("nullable")
-            .Resolve(c => new NullableSchemaType { Data = c.Source as ExampleContext });
+            .Resolve(c => new DataModel { Data = c.Source as ExampleContext });
         query.Field<NonNullableSchemaType>("nonNullable")
-            .Resolve(c => new NonNullableSchemaType { Data = c.Source as ExampleContext });
+            .Resolve(c => new DataModel { Data = c.Source as ExampleContext });
 
         Query = query;
     }
 }
 
-public class NullableSchemaType : ObjectGraphType<NullableSchemaType>
```

Comment:
Ah, even one more place.

Reply:
These are tests, after all - would they really be coded like this for production use? I highly doubt it — running all the graph constructor code for a data model? Seems highly unlikely. 

Could be though - I’m not sure.

Reply:
Tests, yes. We ourselves write a code that should not be written.